### ISSUE TITLE
Hologram click command won't work with Model Engine

### ISSUE THREAD

**Issue by Igzak** [Zrips/CMI#7289]
### Description of Issue

The hologram click command won't work when the Model Engine is installed. Also, hover particles disappear with ModelEngine, something maybe cancel events

### Version Information

```shell
[11:11:31 INFO]: --------------------------------------------------
[11:11:31 INFO]: Версия плагина CMI: 9.2.4.4 BungeeCord CMIB  SqLite
[11:11:31 INFO]: CMILib: 1.2.3.6 
[11:11:31 INFO]: Версия сервера: petal 1.19.2-R0.1-SNAPSHOT
[11:11:31 INFO]: CMI Экономика: Включено Версия Vault: 1.7.3-CMICMI Чат: Включено 
[11:11:31 INFO]: Модули -> 55 включено 2 отключено: tablist, namePlates
[11:11:31 INFO]: --------------------------------------------------
```

### Relevant Plugins

Vault-1.7.3

### Agreements

- [X] My server is supported by CMI.
- [X] My version of CMI at the time of this report is up to date.
- [X] I have searched the github and asked around before making this report.

**Comment by Zrips** [comment]
Can't really test it or debug as its premium plugin. Is old click method still works? Technically new hover effect should not be effected by other plugins as its using player location to calculate needed information based on hologram location. This could only be caused if Model Engine plugin cancels packets.

**Comment by MrCrazys** [comment]
The same happens with me.

**Comment by Igzak** [comment]
Hower effect works good but clicking on old or new method won't work

**Comment by MrCrazys** [comment]
> The Hower effect works good but clicking on the old or new method won't work, there are a lot of requests in ModelEngine discord to fix it but they ignore it for a long time

Yeah, I made a ticket towards the end of September. They still haven’t fixed it.
